Controllers/AccountController.cs

```diff
@@ -0,0 +1,67 @@
+using System.Net;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using baseledger_replicator.BusinessLogic.Auth.Commands;
+using baseledger_replicator.DTOs.Auth;
+using baseledger_replicator.Models;
+
+namespace baseledger_replicator.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AccountController : BaseController
+{
+    private readonly UserManager<IdentityUser> _userManager;
+    private readonly BaseledgerReplicatorContext _baseledgerReplicatorContext;
+    private readonly ILogger _logger;
+
+    public AccountController(UserManager<IdentityUser> userManager, BaseledgerReplicatorContext baseledgerReplicatorContext, ILogger<AccountController> logger)
+    {
+        _userManager = userManager;
+        _baseledgerReplicatorContext = baseledgerReplicatorContext;
+        _logger = logger;
+    }
+    // TODO: Remove this endpoint after testing
+
+    /// <summary>
+    /// Registers a new user
+    /// </summary>
+    /// <param name="registerUserDto"> Contains the register user dto</param>
+    /// <response code="200">Successful registration</response>
+    /// <response code="400">If an error occured during registration</response>
+    [HttpPost("register")]
+    public async Task<IActionResult> RegisterUserAsync([FromBody] RegisterUserDto registerUserDto)
```

Comment:
also authorize registration as it will be always performed  by an existing user.